fenix/app/src/main/java/org/mozilla/fenix/shopping/ReviewQualityCheckFragment.kt

```diff
@@ -33,6 +35,20 @@ class ReviewQualityCheckFragment : BottomSheetDialogFragment() {
             middleware = ReviewQualityCheckMiddlewareProvider.provideMiddleware(
                 settings = requireComponents.settings,
                 browserStore = requireComponents.core.store,
+                openLinkInNewTab = { link ->
+                    (requireActivity() as HomeActivity).openToBrowserAndLoad(
+                        searchTermOrURL = link,
+                        newTab = true,
+                        from = BrowserDirection.FromReviewQualityCheck,
+                    )
+                },
+                openLinkInSelectedTab = { link ->
+                    (requireActivity() as HomeActivity).openToBrowserAndLoad(
+                        searchTermOrURL = link,
+                        newTab = true,
```

Comment:
this should be false in this lambda, right?
```suggestion
                        newTab = false,
```